src/counts.jl

```diff
@@ -9,53 +9,90 @@
 typealias IntUnitRange{T<:Integer} UnitRange{T}
 
 #### functions for counting a single list of integers (1D)
+"""
+    addcounts!(r, x, levels[, wv])
 
+Fill the array `r` with the number of occurrences in `x` of
```

Comment:
Hmm, after reading the `init` code, I realize this function does not "fill `r`" but "adds to `r`".